.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release packages
+
+on:
+  push:
+    tags:
+      - '*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  basepkgs:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Vanilla amd64
+            board: amd64
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: Vanilla aarch64
+            board: aarch64
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+          - name: Vanilla rpi4
+            board: rpi4
+            parallel: 2
+            image: basefocal:20.04.0
+          - name: Vanilla odroid-xu4
+            board: odroid-xu4
+            parallel: 2
+            image: qnnpack:20.04.0
+          - name: Vanilla lx2160
+            board: lx2160
+            parallel: "$(nproc)"
+            image: basefocal:20.04.0
+
+    name: ${{ matrix.name }}
+    uses: klepsydra-technologies/kpe-ghworkflows/.github/workflows/generate_release.yml@v2.0.6
```

Comment:
Remove release.yml? (Not in kpsr-core.)

Reply:
Not sure if we have a mechanism for storing packages for public repos. One thing that is certain is the kpe-ghworkflows cannot be used for it. @tomasz3dk or @pablogh-2000 can confirm if we should delete release.yml or not.

Reply:
I agree. If we can not use kpe-ghworflows, we should remove it.

Reply:
file removed.